src/kit.ts

```diff
@@ -526,11 +561,31 @@ async function collectDevBatVars(devbat: string, args: string[], major_version:
 
 /**
  * Platform arguments for VS Generators
+ * For example, x86 and amd64_x86 will generate -A win32
  */
-const VsArchitectures: {[key: string]: string} = {
+const genPlatformFromVsHostTargetArchs: {[key: string]: string} = {
+  x86: 'win32',
```

Comment:
If the key is a string, it would be best to add the keys with string syntax:

```ts
  'x86': 'win32',
```

Reply:
I tried but I get "Unnecessarily quoted property 'x86' found."